src/Config.php

```diff
@@ -1079,15 +1079,19 @@ public function showFormUserPrefs($data = [])
 
         echo "</tr>";
 
-        echo "<tr class='tab_bg_2'>";
         if ($oncentral) {
-            echo "<td><label for='dropdown_use_flat_dropdowntree$rand'>" . __('Display the complete name in tree dropdowns') . "</label></td><td>";
+            echo "<tr class='tab_bg_2'>";
+            echo "<td><label for='dropdown_use_flat_dropdowntree$rand'>" . __('Display the tree dropdown complete name in dropdown inputs') . "</label></td><td>";
             Dropdown::showYesNo('use_flat_dropdowntree', $data["use_flat_dropdowntree"], -1, ['rand' => $rand]);
             echo "</td>";
-        } else {
-            echo "<td colspan='2'>&nbsp;</td>";
+
+            echo "<td><label for='dropdown_use_flat_dropdowntree_on_search_result$rand'>" . __('Display the tree dropdown complete name on search result') . "</label></td><td>";
```

Comment:
the sentence seems odd.
I suggest: "Display the complete name of tree dropdown in search results" but an advice from @cconard96 would be appreciated :)

Reply:
"Display the complete name of tree dropdown in search results" makes a lot more sense to me.